freqtrade/rpc/local_rest_server.py

```diff
@@ -0,0 +1,83 @@
+import threading
+import logging
+import json
+
+from flask import Flask, request
+from flask_restful import Resource, Api
+from json import dumps
+from freqtrade.rpc.rpc import RPC
+
+
+logger = logging.getLogger(__name__)
+
+
+class Daily(Resource):
+    # called by http://127.0.0.1:/daily?timescale=7
+    # where 7 is the number of days to report back with
+
+    def __init__(self, freqtrade) -> None:
+        """
+        Initializes all enabled rpc modules
+        :param freqtrade: Instance of a freqtrade bot
+        :return: None
+        """
+        self.freqtrade = freqtrade
+        self._config = freqtrade.config
+
+
+    def get(self):
+        timescale = request.args.get('timescale')
+        logger.info("LocalRPC - Daily Command Called")
+        timescale = int(timescale)
+
+        (error, stats) = RPC.rpc_daily_profit(self, timescale,
+                                          self._config['stake_currency'],
+                                          self._config['fiat_display_currency']
+                                          )
+        if error == False:
+            stats = dumps(stats, indent=4, sort_keys=True, default=str)
+            return stats
+        else:
+            json.dumps(error)
+            return error
+
+
+class LocalRestSuperWrap(RPC):
```

Comment:
I think all app.route decorated methods should also be in this class (in that case they have access to self._rpc_* methods). In this PR those methods are called as if they are static.

Reply:
done.